### PR TITLE
Add script to convert `um_env.py` files to yaml

### LINKED ISSUE #1: Add script for converting UM um_env.py configuration files to YAML
ESM1.5 configurations currently use a `um_env.py` python file, containing a dictionary for setting environment variables. In the [next version of payu](https://github.com/payu-org/payu/pull/459), this will be replaced by a YAML file. 

We should add a small script for converting  a `um_env.py` file into a `um_env.yaml` file, so that its simple to modify an already existing configuration to run with the newest version of payu.

### PR DESCRIPTION
**Pull request by blimlim**
Closes #1 

This pull request adds a small script which loads in a `um_env.py` configuration file, and saves the resulting dictionary to a `yaml` file.

Running a simulation with a `um_env.yaml` file produced by the script results in the same values for all the UM environment variables compared to a run using a `um_env.py` file, other than `DATAW` which is different for every simulation. 

Any suggested changes are welcome!

### CODE REVIEW

File: config-files/UM/um_env_to_yaml.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#   python um_env_to_yaml.py <input_filepath> [--ofile <output_filepath>]
+#
+# If --ofile is not specified, it defaults to the same directory as  <input_filepath>
+# with filename "um_env.yaml"
+#
+# Contact:
+#   Spencer Wong <spencer.wong@anu.edu.au>
+#
+# Dependencies:
+#    pyyaml
```
Comment (CodeGat) [inline config-files/UM/um_env_to_yaml.py:19]: It's been a while...is this a standard library? If it isn't should there be a `requirements.txt` that people can `pip install` these dependencies?

**Review by CodeGat — COMMENTED** [review]
Looks good to me. Just one comment.

File: config-files/UM/um_env_to_yaml.py (hunk 1 above)
Comment (blimlim) [reply config-files/UM/um_env_to_yaml.py:19]: Good idea! I don't think it is a standard library – I've added in a `requirements.txt` file.

**Review by CodeGat — APPROVED** [review]

**Comment by CodeGat** [comment]
Added some branch protections, as well

**Pull request merged**